package.json

```diff
@@ -22,5 +21,9 @@
   "homepage": "https://github.com/quizizz/service-communication-wrapper#readme",
   "dependencies": {
     "axios": "^0.27.2"
```

Comment:
let's also take this opportunity to upgrade axios to the latest version - 1.6.2
but we need to be careful and take care of any breaking type changes
[https://github.com/axios/axios/blob/v1.x/CHANGELOG.md](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)